versioned/storage/cache/src/main/java/org/projectnessie/versioned/storage/cache/CaffeineCacheBackend.java

```diff
@@ -31,24 +36,30 @@
 abstract class CaffeineCacheBackend implements CacheBackend {
 
   public static final int JAVA_OBJ_HEADER = 32;
+  public static final String CACHE_NAME = "nessie-objects";
 
   static ImmutableCaffeineCacheBackend.Builder builder() {
     return ImmutableCaffeineCacheBackend.builder();
   }
 
   abstract long capacity();
 
+  @Nullable
+  @jakarta.annotation.Nullable
+  abstract MeterRegistry meterRegistry();
+
   @Value.Derived
   Cache<CacheKey, byte[]> cache() {
-    // IMPORTANT!
-    // When changing the configuration of the Caffeine cache, make sure to run the
-    // _native_ Quarkus tests and adopt the `@ReflectionConfig` in
-    // org.projectnessie.quarkus.providers.PersistProvider.
-    return Caffeine.newBuilder()
-        .maximumWeight(capacity())
-        .recordStats()
-        .weigher(this::weigher)
-        .build();
+    Caffeine<CacheKey, byte[]> cacheBuilder =
+        Caffeine.newBuilder().maximumWeight(capacity() * 1024L * 1024L).weigher(this::weigher);
```

Comment:
Could be good to add some javadocs to explain which unit we expect `capacity()` to operate on.

Reply:
done